yci/skills/_shared/customer-isolation/scripts/extract-paths.py

```
@@ -0,0 +1,171 @@
+    _extract_from_tool_input(tool_name, tool_input, cwd, paths)
+
+    for path in paths:
+        print(path)
+
+
+if __name__ == "__main__":
+    try:
+        main()
+    except Exception:
+        # Never raise past the top level — always exit 0.
+        pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)